shared/fs/nav-header/main-banner/index.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react'
 import * as Kb from '../../../common-adapters'
 import * as Types from '../../../constants/types/fs'
-import {isMobile} from '../../../constants/platform'
+import * as Styles from '../../../styles'
 
 // /*
```

Comment:
if this isn't true anymore and hasn't been true for a while, maybe we should just delete this comment.

Reply:
Thanks; this mechanism was moved into nav-header/mobile-header.tsc but I forgot to delete this. Will do